dataactvalidator/config/sqlrules/c9_award_financial.sql

```diff
@@ -1,14 +1,14 @@
 SELECT afa.row_number, afa.fain, afa.uri
 FROM award_financial_assistance AS afa
 WHERE afa.submission_id = {0}
-	AND afa.federal_action_obligation > 0
+	AND CAST(afa.federal_action_obligation as numeric) > 0
```

Comment:
If `federal_action_obligation` is described as a text data type in the schema and we do, in fact, get text input, this will break. 

Looks like Postgres doesn't have an `ISNUMERIC` function, so we'd have to futz with this. Ultimately, I don't think it's worth holding up the deploy for. I'd suggest:
1. adding a note to the C9 story on the board (I'll do that)
2. checking in the schema team--it doesn't make sense to have a numeric rule on a field that's defined as text.